setup.py

```diff
@@ -27,14 +27,68 @@
     "rsa>=3.1.4,<5",
 )
 
+# TODO(https://github.com/googleapis/google-auth-library-python/issues/1440): Unit test fails with
+#  `No module named 'cryptography.hazmat.backends.openssl.x509' for Python 3.7``.
+cryptography_base_require = [
+    "cryptography>=38.0.3",
+    "cryptography < 39.0.0; python_version < '3.8'",
+]
+
+requests_extra_require = ["requests >= 2.20.0, < 3.0.0.dev0"]
```

Comment:
Remove `.dev0`. See https://github.com/googleapis/google-cloud-python/issues/13585. Applies throughout

```suggestion
requests_extra_require = ["requests >= 2.20.0, < 3.0.0"]
```